aspnetcore/blazor/get-started.md

```diff
@@ -24,8 +24,9 @@ In a few steps, get started with Blazor:
 
 1. Follow the guidance for your choice of tooling:
 
+   <!-- markdownlint-disable MD023 -->
```

Comment:
Hi @nschonni ... Please revert the changes to this file. I don't want the markdownlint disable rule in here because they clutter the content and will confuse community contributors. We have enough cryptic rules and guidelines on the repo as it is.

@scottaddie Should the linter issue here be raised elsewhere? I think markdownlint disable rules sprinkled in all over the place is going to end in tears :cry:. IMHO, either the linter should be fixed or ignored.

Reply:
Another factor on this topic, too ... I'm about to merge a bunch of changes to it. It's best if I pick up this work on that PR. Otherwise, one or the other of us is going to create a :boom:.

Reply:
I'm in agreement that we don't want to disable rules inline. All rules configuration should be handled in the config file.

Reply:
I rebased out this change.
I do think it's better to suppress rules inline when there are only a few exceptions throughout the codebase